experiments/IB_collab_mode.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+
+#Test script for writing new experiment!
+
+# write an experiment that creates a new control program.
+import os
+import sys
+import copy
+
+BOREALISPATH = os.environ['BOREALISPATH']
+sys.path.append(BOREALISPATH)
+
+from experiment_prototype.experiment_prototype import ExperimentPrototype
+import experiments.superdarn_common_fields as scf
+from experiment_prototype.decimation_scheme.decimation_scheme import DecimationScheme, DecimationStage, create_firwin_filter_by_attenuation
+
+
+def create_15km_scheme(): 
+    """
+    Frankenstein script by Devin Huyghebaert for 15 km range gates with a special ICEBEAR collab mode
+
+    previous comment:
+    Create four stages of FIR filters and a decimation scheme. Returns a decimation scheme of type DecimationScheme. 
+    This filter will have a wider receive bandwidth than the previous.
+    Pasha recommends a 10kHz bandwidth for the final stage. I believe there will be aliasing caused by this but 
+    perhaps the concern is not critical because of the small bandwidth overlapping. I will test this anyways.
+    :return DecimationScheme: a decimation scheme for use in experiment.
+    """
+
+    rates = [5.0e6, 500.0e3, 100.0e3, 50.0e3] # last stage 50.0e3/3->50.0e3
+    dm_rates = [10, 5, 2, 5] # third stage 6->2
+    transition_widths = [150.0e3, 40.0e3, 15.0e3, 1.0e3] #did not change
+    cutoffs = [20.0e3, 10.0e3, 10.0e3, 5.0e3] # bandwidth is double this.  Did not change
+    ripple_dbs = [150.0, 80.0, 35.0, 8.0] # changed last stage 9->8
+    scaling_factors = [10.0, 100.0, 100.0, 100.0] # did not change
+    all_stages = []
+
+    for stage in range(0,4):
+        filter_taps = list(scaling_factors[stage] * create_firwin_filter_by_attenuation(rates[stage], transition_widths[stage], cutoffs[stage], ripple_dbs[stage]))
+        all_stages.append(DecimationStage(stage, rates[stage], dm_rates[stage], filter_taps))
+
+    return (DecimationScheme(5.0e6, 10.0e3, stages=all_stages)) # changed from 10e3/3->10e3
```

Comment:
the first two arguments can be grabbed from the `stage` list above (I think that the first item in the list is the USRP sample rate. in general, and the last is the output sample rate * final dm rate, in general).
```suggestion
    return (DecimationScheme(rates[0], rates[-1]/dm_rates[-1], stages=all_stages)) # changed from 10e3/3->10e3
```